hack/jenkins/minishift_linux_integration_tests_kvm.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# This script runs the integration tests on a Linux machine for the KVM Driver
+
+# The script expects the following env variables:
+# MINIKUBE_LOCATION: GIT_COMMIT from upstream build.
+# COMMIT: Actual commit ID from upstream build
+# EXTRA_BUILD_ARGS (optional): Extra args to be passed into the minikube integrations tests
+# access_token: The Github API access token. Injected by the Jenkins credential provider.
+
+set -e
+
+OS_ARCH="linux-amd64"
+VM_DRIVER="kvm"
+JOB_NAME="Minishift-Linux-KVM"
+BINARY=$(pwd)/out/minishift
+EXTRA_FLAGS="--show-libmachine-logs"
+
+# Download files and set permissions
+mkdir -p out/
+
+curl -L http://artifacts.ci.centos.org/minishift/minishift/master/latest/linux-amd64/minishift -o out/minishift
```

Comment:
would be great to at least do a `minishift version` once, as latest is not a good indication of the actual version we used to run the tests with. @praveenkumar WDYT?